tests/integration/runner

```diff
@@ -407,8 +406,14 @@ if __name__ == "__main__":
     if args.analyzer:
         use_analyzer = "-e CLICKHOUSE_USE_NEW_ANALYZER=1"
 
-    pytest_opts = " ".join(args.pytest_args).replace("'", "\\'")
-    tests_list = " ".join(args.tests_list)
+    # NOTE: since pytest options is in the argument value already we need to additionally escape '"'
+    pytest_opts = " ".join(
```

Comment:
@azat this broke sending extra arguments to pytest like `-k` because everything is put under the same quote and parsed as test name.

Reply:
It should work, maybe you are using old env variable name? The new one is `PYTEST_ADDOPTS`

Reply:
did you try running an integration test using runner and filtering, e.g. `test_keeper_back_to_back -k test_concurrent_watches`?

AFAIS, the entire `pytest_args` are quoted.